### ISSUE TITLE
KeyError: 'LocationConstraint'

### ISSUE THREAD

**Issue by exarkun** [WhoopInc/mkwheelhouse#1]
An attempted to make a wheelhouse fails:

```
Traceback (most recent call last):
  File "bin/mkwheelhouse", line 9, in <module>
    load_entry_point('mkwheelhouse==0.2.2', 'console_scripts', 'mkwheelhouse')()
  File "lib/python2.7/site-packages/mkwheelhouse.py", line 106, in main
    index_url = bucket.resource_url('index.html')
  File "lib/python2.7/site-packages/mkwheelhouse.py", line 35, in resource_url
    return os.path.join(self.endpoint().host, self.name, resource)
  File "lib/python2.7/site-packages/mkwheelhouse.py", line 29, in endpoint
    return self.s3_service.get_endpoint(self.region())
  File "lib/python2.7/site-packages/mkwheelhouse.py", line 25, in region
    self._region = response_data['LocationConstraint']
KeyError: 'LocationConstraint'
```

**Comment by benesch** [comment]
You have to create the bucket first and make sure your AWS key has permission to execute a GetBucketLocation query. Sorry that this is a) undocumented, and b) a terrible error message.

As soon as boto supports py3k, I'll port our hand-rolled API requests over, which should give us much better error messages.

**Comment by exarkun** [comment]
This happened on Python 2.7, for what it's worth.

**Comment by benesch** [comment]
Fixed in 1.0!
